cypress/e2e/checkout.cy.js

```diff
@@ -1,11 +1,47 @@
 /// <reference types='cypress' />
+import ContactFormPageObject from '../support/pages/contactForm.pageObject';
+import HomeAndCataloguePageObject
+  from '../support/pages/homeСatalogue.pageObject';
+import faker from 'faker';
 
-describe('', () => {
-  before(() => {
+const contactForm = new ContactFormPageObject();
+const homePage = new HomeAndCataloguePageObject();
 
+const testData = {
+  name: faker.name.firstName(),
+  addedProductMessage: 'Product added',
+  country: faker.random.word(),
+  city: faker.random.word(),
```

Comment:
may generate random word not a city or country, it is better to use random city/country over here to be close to real data